subworkflows/local/initialize_annotation_cache/main.nf

```diff
@@ -0,0 +1,57 @@
+//
+// INITIALIZE ANNOTATION CACHE
+//
+
+// Initialize channels based on params or indices that were just built
+// For all modules here:
+// A when clause condition is defined in the conf/modules.config to determine if the module should be run
+// Condition is based on params.step and params.tools
+// If and extra condition exists, it's specified in comments
```

Comment:
```suggestion
// If an extra condition exists, it's specified in comments
```

Reply:
not fixing that here, as it's a copy paste and I'm sure it's in every other subworkflows, I'll make a separate PR for it